packages/ag-grid-theme/css/_salt-ag-theme-default-params.scss

```diff
@@ -13,9 +12,6 @@ $salt-ag-theme-default-params: (
       "Open Sans",
       sans-serif,
     ),
-  icon-font-family: $salt-ag-theme-icon-font-family,
```

Comment:
Overriding icons like this means we cannot mix and match between the default icons that the jpmuitk-icons font icons